### ISSUE TITLE
Uncaught TypeError: Cannot read property 'append' of undefined

### ISSUE THREAD

**Issue by stramel** [atom/settings-view#321]
### Steps to Reproduce

1. Update Packages from settings-view

**Atom Version**: 0.169.0-1bc9c2e
**System**: linux 3.13.0-43-generic
**Thrown From**: [settings-view](https://github.com/atom/settings-view) package, v0.165.0

### Stack Trace

Uncaught TypeError: Cannot read property 'append' of undefined

```
At /usr/local/share/atom/resources/app/node_modules/settings-view/lib/installed-packages-panel.js:132

TypeError: Cannot read property 'append' of undefined
  at /usr/local/share/atom/resources/app/node_modules/settings-view/lib/installed-packages-panel.js:132:36
  at /usr/local/share/atom/resources/app/node_modules/emissary/lib/emitter.js:155:30
  at Array.forEach (native)
  at PackageManager.module.exports.Emitter.emitSlow (/usr/local/share/atom/resources/app/node_modules/emissary/lib/emitter.js:154:47)
  at PackageManager.module.exports.Emitter.emit (/usr/local/share/atom/resources/app/node_modules/emissary/lib/emitter.js:107:30)
  at PackageManager.module.exports.PackageManager.emitPackageEvent (/usr/local/share/atom/resources/app/node_modules/settings-view/lib/package-manager.js:368:19)
  at /usr/local/share/atom/resources/app/node_modules/settings-view/lib/package-manager.js:242:19
  at exit (/usr/local/share/atom/resources/app/node_modules/settings-view/lib/package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (/usr/local/share/atom/resources/app/src/buffered-process.js:63:47)
  at /usr/local/share/atom/resources/app/src/buffered-process.js:70:18
  at Socket.<anonymous> (/usr/local/share/atom/resources/app/src/buffered-process.js:150:18)
  at Socket.emit (events.js:129:20)
  at Pipe.close (net.js:469:12)

```

### Commands

```
  2x -0:00.0 core:close (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "welcome",
      "open-on-github",
      "background-tips",
      "language-xml",
      "package-generator",
      "release-notes",
      "selection-count",
      "styleguide",
      "linter-jscs",
      "atom-spotify",
      "autocomplete-paths",
      "atom-color-highlight"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ],
    "projectHome": "~/Projects/foundationui"
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.1.0
atom-beautify, v0.21.2
atom-jasmine, v0.4.0
autoclose-html, v0.13.0
autocomplete-plus, v1.1.0
autocomplete-snippets, v0.3.2
editorconfig, v0.2.3
file-icons, v1.4.4
highlight-line, v0.9.2
highlight-selected, v0.7.0
linter, v0.9.1
linter-bootlint, v0.0.3
linter-jshint, v0.1.0
linter-less, v0.3.1
monokai, v0.9.0
pain-split, v1.1.2
project-manager, v1.14.1
project-palette-finder, v2.4.7

# Dev
No dev packages
```

/cc @atom/core

**Comment by kevinsawicki** [comment]
Dupe of https://github.com/atom/settings-view/issues/317 which was fixed and released in 0.169

**Comment by stramel** [comment]
Ah! Must have just missed this release this morning! Just updated, thanks!
